packages/zero-client/src/client/make-replicache-mutators.test.ts

```diff
@@ -217,29 +215,28 @@ describe('extendReplicacheMutators', () => {
     const context = sharedContext;
     const mutateObject: Record<string, unknown> = {};
 
-    const defineTypedMutator = defineMutatorWithType<
+    const incrementMutator = defineMutator<
+      undefined,
       Schema,
       {counter: number}
-    >();
+    >(({ctx}: {ctx: {counter: number}}) => {
+      ctx.counter++;
+      return Promise.resolve();
+    });
 
-    const incrementMutator = defineTypedMutator(
+    const getMutator = defineMutator<undefined, Schema, {counter: number}>(
       ({ctx}: {ctx: {counter: number}}) => {
-        ctx.counter++;
+        // Read the counter value
+        void ctx.counter;
         return Promise.resolve();
       },
     );
 
-    const getMutator = defineTypedMutator(({ctx}: {ctx: {counter: number}}) => {
-      // Read the counter value
-      void ctx.counter;
-      return Promise.resolve();
-    });
-
-    const defineTypedMutators = defineMutatorsWithType<
+    const mutators = defineMutators<
+      {increment: typeof incrementMutator; get: typeof getMutator},
```

Comment:
I am going to address this added type goop in a future PR. `defineMutators` should not need explicit typing - will fix the contravariance issues.

Reply:
Yeah. This seems like a blocker at the moment.

Thanks.